docker-compose.yml

```diff
@@ -23,7 +23,9 @@ services:
     image: brightid/contexts_updater
 
   db:
-    build: db
+    build:
+      context: ./db
+      dockerfile: Dockerfile.with-backup
```

Comment:
Is the idea here for the user to specify either `Dockerfile` or `Dockerfile.with-backup`? If so, I think the default should be `Dockerfile`